src/tools/jedutil.cpp

```diff
@@ -2167,9 +2206,22 @@ static pal_data paldata[] = {
 		config_pal20x10_pins,
 		nullptr,
 		nullptr},
-	/*{"PAL22V10", 0, NULL, 0, NULL, 0, NULL, NULL, NULL, NULL},
-	{"GAL20V8A", 0, NULL, 0, NULL, 0, NULL, NULL, NULL, NULL},
+	/*{"GAL20V8A", 0, NULL, 0, NULL, 0, NULL, NULL, NULL, NULL},
 	{"GAL22V10", 0, NULL, 0, NULL, 0, NULL, NULL, NULL, NULL},*/
+    {"PAL22V10", 5828,
```

Comment:
Indentation is off on this line

Reply:
I had spaces in there instead of a tab character.  Can I fix this with my next commit?  I'm working on adding the ability to list the devices that match a JED file based on the fuse count.